### PR TITLE
Added custom form size section

### PR DESCRIPTION
**Pull request by olcayertas**
Showing forms in a subview of the view controller.
Original content was here: https://github.com/xmartlabs/Eureka/issues/723

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Made with ❤️ by [XMARTLABS](http://xmartlabs.com). This is the re-creation o
   + [Multivalued sections]
   + [Validations]
   + [Swipe Actions]
+* [Custom form size]
```
Comment (mats-claassen) [inline README.md]: I would put this under `Usage`. I think it is not a main item in the Readme like `Usage` or `Custom Rows`.

File: README.md [hunk 2]
```diff
@@ -632,6 +633,14 @@ let row = TextRow() {
 Swipe Actions need `tableView.isEditing` be set to `false`. Eureka will set this to `true` if there is a MultivaluedSection in the form (in the `viewWillAppear`).
 If you have both MultivaluedSections and swipe actions in the same form you should set `isEditing` according to your needs.
 
+## Custom form size
+
+Forms will span the entire view controller when you extend FormViewController. If you want to show your form in a subview of your view controller follow below steps:
```
Comment (mats-claassen) [inline README.md]: `follow below steps` -> follow the steps below

File: README.md [hunk 3]
```diff
@@ -632,6 +633,14 @@ let row = TextRow() {
...
+
+1. Add your UITableView to the view in your storyboard. Make sure the table view is styled as grouped rather than plain.
+2. Drag from the tableView circle in the Outlets panel in the Connections Inspector. This outlet is provided by Eureka's FormViewController so there is no need to override it or create a new @IBOutlet for the table view.
```
Comment (mats-claassen) [inline README.md]: I am  not convinced by the first sentence. I would say something like `Select the view controller and drag the tableView outlet from the Outlets panel to your UITableView on the storyboard.`

**Comment by olcayertas** [comment]
Thank you @mats-claassen.

**Comment by mtnbarreto** [comment]
Hi Guys!
Sometimes it would be better to add a ContainerView that  fits the desired size.  This should be addressed in the doc. 

> https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html

File: README.md (hunk 1 above)
Comment (olcayertas) [reply README.md]: OK, thanks!

**Pull request closed without merging**